packages/@ourworldindata/grapher/src/timeline/TimelineComponent.tsx

```diff
@@ -428,6 +428,26 @@ export class TimelineComponent extends React.Component<TimelineComponentProps> {
         }
     }
 
+    @action.bound private handleYearKeyboardNavigation(
+        key: string,
+        currentTime: number
+    ): number {
```

Comment:
The keyboard behavior for when hitting ArrowUp and then reaching the endTime is a bit confusing, as you can see in the video ([chart link](http://staging-site-discrete-timeline/grapher/life-expectancy?time=earliest..1800)): You can reach the endTime, but then when you hit enter it won't let you do just that.

https://github.com/user-attachments/assets/7553079d-dfa7-4574-8ec4-9bcfeb59b1f5

I can see two possible ways to make that more intuitive:
- (1) disallow increasing to 1800 in the first place, since `allowHandlesOnSameTime` is disabled
- (2) allow increasing beyond 1800 (as it is very much possible when entering numbers), and then possibly also "skip" over 1800 since that's an impossible option